packages/client/src/InfluxDBClient.ts

```diff
@@ -135,23 +161,32 @@ export default class InfluxDBClient {
    *
    * @param query - The query string.
    * @param database - The name of the database to query.
-   * @param queryType - The type of query (default: 'sql').
-   * @param namedParams - for sql queries parameters to be used
+   * @param queryOptions - The options for the query (default: \{ type: 'sql' \}).
+   * @example
+   * ```typescript
+   *    client.query('SELECT * from testData', 'CI_TEST', {
```

Comment:
Maybe we can use something more "production" in examples like:

```suggestion
   *    client.query('SELECT * from cpu', 'performance_db', {
```

Reply:
I've updated examples in TS doc to reflect production like values